agent/command/results_native.go

```diff
@@ -128,41 +119,5 @@ func (c *attachResults) Execute(ctx context.Context,
 		return errors.Wrapf(err, "reading report file '%s'", reportFileLoc)
 	}
 
-	if err := c.sendTestLogs(ctx, conf, logger, comm, &nativeResults); err != nil {
```

Comment:
I'm confused: if we're not going to send these anymore does that mean that logs will only be sent if they're in the special directory? Does that mean we need to get everyone to start using the new way first?

Reply:
Resmoke does not add its logs to the test results inline but rather provides the Logkeeper URL links. We also do not document this feature on the user-facing docs so I think it is safe to remove.